content/md/cartridge-configuration/06-mapping-configuration.md

```diff
@@ -119,6 +119,16 @@ Please, note that each PIM attribute is prefixed with the `akeneo_` label in Sal
 Note: If you don't want to use this feature, please leave a space between curly brackets ({ }).
 :::
 
+## How to exclude Attribute groups from the import process?
+
+More and more users need to exclude a whole attribute group from their import process. Most of the time, you need that feature because you have PIM technical attributes you don't want to be copied in Magento. To do so, please use [**the Connection rights**](https://help.akeneo.com/pim/serenity/articles/manage-your-connections.html#configure-the-connection-user-group) to exclude this attribute group(s) from the connector access right and then to avoid to import the group(s).
```

Comment:
```suggestion
More and more users need to exclude a whole attribute group from their import process. Most of the time, you need that feature because you have PIM technical attributes you don't want to be copied in SFCC. To do so, please use [**the Connection rights**](https://help.akeneo.com/pim/serenity/articles/manage-your-connections.html#configure-the-connection-user-group) to exclude this attribute group(s) from the connector access right and then to avoid to import the group(s).
```